internal/pkg/http/client/pool.go

```diff
@@ -99,7 +99,15 @@ func (p *Pool) StartAndWait() error {
 func (p *Pool) wait() error {
 	defer close(p.responsesChan)
 	defer close(p.requestsChan)
-	return p.workers.Wait()
+	defer func() { p.finished = true }()
+	if err := p.workers.Wait(); err != nil {
+		p.log("stopped on error: %s | %s", err, time.Since(p.startTime))
+		return err
+	}
+	if p.requestsSentCount.Get() > 0 {
+		p.log("all done | %s", time.Since(p.startTime))
+	}
```

Comment:
Presunute logovanie hlasky z goroutine do `wait`.

--------

Padalo to v nedavno pridanom test `TestErrorInSubRequest_ParallelRequests`.

Loguje sa tam z dvoch paralelnych requestov, co je osetrene:
https://github.com/keboola/keboola-as-code/blob/67c7e4c4368d262cfd1716e29a69e815b171ee43/internal/pkg/http/client/pool_test.go#L226-L231

No loguje sa aj zo samotneho http pool:
https://github.com/keboola/keboola-as-code/blob/67c7e4c4368d262cfd1716e29a69e815b171ee43/internal/pkg/http/client/pool.go#L118

A tam nebolo garantovane, ze sa tato hlaska zaloguje pred navratom zo `StartAndWait`.

A pri tomto asserte sa citaju `InfoMessages`, ale zaroven sa logovala este ^^^ debug hlaska (`all done`):
https://github.com/keboola/keboola-as-code/blob/67c7e4c4368d262cfd1716e29a69e815b171ee43/internal/pkg/http/client/pool_test.go#L247

![image](https://user-images.githubusercontent.com/19371734/155678532-03923074-45c8-4f93-a32a-1285ed268e6e.png)


Na fungovanie CLI to nemalo ziadny vplyv, iba niekedy spadol test.